infra/aws/README.md

```diff
@@ -0,0 +1,39 @@
+## AWS CI Infra
+
+This setup is for the Buildkite Elastic CI Stack on AWS.
+Services used:
+- VPC with 4 subnets (each subnet carries 16384 IP addresses)
+- 4 CloudFormation stacks: `small-cpu-queue`, `cpu-queue`, `gpu-1-queue`, and `gpu-4-queue` which has corresponding Buildkite agent queue
+    - `small_cpu_queue` is for running small CPU jobs like bootstrapping CI build or building documentation
+    - `cpu_queue` is for heavier CPU jobs like building vLLM Docker image or running CPU tests
+    - `gpu_1_queue` is for running all GPU tests that require 1 GPU
+    - `gpu_4_queue` is for running all GPU tests that require 2 or 4 GPUs
+- Each stack has access to write/read from ECR public repo (that stores vLLM Docker images) and AWS Secrets Manager that stores Huggingface token used for testing.
+
+## Setup with Terraform
+
+You need to add the 2 following credentials:
+- vLLM AWS access key ID & secret access key 
+- Buildkite API token (with access to vLLM org and with GraphQL API Access enabled) `export BUILDKITE_API_TOKEN=....`
+
+To set up, start with initalizing:
+```bash
+terraform init
```

Comment:
so... should we use an s3 bucket as a backend for saving the states?

or after you applied on one machine, how does one (or another person) make changes on another machine?

Reply:
will leave you to learn a bit on this part, but we can talk more about this on the details maybe next monday.

I would like to try myself running the `terraform init` and `terraform plan` on my machine on monday, to make sure the workflow works.

Reply:
yes the state file should be stored on s3

Reply:
Updated to use remote state and also updated README so that remote state should be loaded before doing `terraform init`